src/mesh/Rope.js

```diff
@@ -52,13 +51,11 @@ export default class Rope extends Mesh
         this.indices = new Uint16Array(points.length * 2);
 
         /**
-         * Tracker for if the rope is ready to be drawn. Needed because Mesh ctor can
-         * call _onTextureUpdated which could call refresh too early.
-         *
+         * refreshes vertices on every updateTransform
          * @member {boolean}
-         * @private
+         * @default true
          */
-        this._ready = true;
+        this.autoUpdate = true;
```

Comment:
Could you explain what `autoUpdate` is necessary as a public API? When would you want to set this to `false`?